src/components/BuySellForm/BuySellPaymentSection/BuySellPaymentSection.tsx

```diff
@@ -26,7 +26,8 @@ const BuySellPaymentSection = ({
     setIsHidden,
 }: TBuySellPaymentSectionProps) => {
     const { isDesktop } = useDevice();
-    const sortedList = sortPaymentMethodsWithAvailability(availablePaymentMethods);
+    const sortedList = sortPaymentMethodsWithAvailability(availablePaymentMethods).filter(p => !p.isAvailable);
+    const disableAdvertiserPaymentMethods = selectedPaymentMethodIds.length === 3;
```

Comment:
sorted list will only display the non available payment methods so users can Add that payment method